src/components/Chat.js

```diff
@@ -201,16 +207,29 @@ class TextComposer extends React.Component {
       <TextComposer.Context.Provider value={context}>
         <StyledTextComposer {...this.props}>
           {children}
+            <StyledSendButton disabled={!context.value} onClick={this._handleSendButton} >{context.value ? "Send": ""}</StyledSendButton>
```

Comment:
Hiding and showing the button should be done using CSS not by changing the text inside it to an empty string - the button is still there even without text.

The simple case: set `display: none` when there's no value
(the nicer case would be to have some CSS gentle animation)